regtests/bin/matrix_ncep

```diff
@@ -63,15 +71,26 @@
     modmpi='hpc-impi/2019.6'
     modnetcdf='netcdf/4.7.4'
     metispath='/work/noaa/marine/ali.abdolali/Source/hpc-stack/parmetis-4.0.3'
+    modjasper='jasper/2.0.22'
+    modzlib='zlib/1.2.11'
+    modpng='png/1.6.35'
+    modhdf5='hdf5/1.10.6'
+    modbacio='bacio/2.4.1'
+    modg2='g2/3.4.1'
+    modw3nco='w3nco/2.4.1'
+    modesmf='esmf/8_1_0_beta_snapshot_47'
   else
     batchq=
   fi
 
 # 1. Set up
    
-  export  np='24'  #number of mpi tasks
-  export  nr='4'   #number of mpi tasks for hybrid
-  export nth='6'   #number of threads
+  export  np='24'     #number of mpi tasks
+  export  npl='440'   #number of mpi tasks for ufs applications and large setups
```

Comment:
440*8 seems excessive for a regression test.  While this is close to the operational set-up which might be important to test on a slower cadence than every commit, for GDAS we use 140 tasks and 4threads which seems like a much more manageable number, especially given that the 1degree ufs 1.1 is also using these task counts.     Is there a reason behind the choice of using --cpus-per-task=${nth} instead of just requesting a certain set of nodes?

Reply:
I'll test the suggested combinations and will report.

Reply:
@JessicaMeixner-NOAA Done